docs/source/system_reference_guide/custom-environments.ipynb

```diff
@@ -38,27 +38,33 @@
     "\n",
```

Comment:
<p>bold or italicize <strong><em>libmamba</em></strong></p><p> </p><p>use a callout box to talk about libmamba</p><pre class="ql-syntax" spellcheck="false">&gt; Note:
text
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
@wildintellect I agree with bold or italicize libmamba, but how would you suggest I change the line:
"We use `conda` with the libmamba solver as a package manager to install, update or remove packages (libraries)."? libmamba is in a callout box even when it is in the middle of the sentence?

Reply:
the callout box should be after the sentence explaining what libmamba is and how you can use classic if you want/need

Reply:
So `conda install --solver=classic -c conda-forge xarray` is in the callout box?

Reply:
@wildintellect Just following up on the above comment